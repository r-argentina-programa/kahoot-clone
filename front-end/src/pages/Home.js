import React from 'react';
import { Link } from 'react-router-dom';
import Button from 'react-bootstrap/Button';

const Home = () => {
  return (
    <React.Fragment>
      <body className="bg-primary">
        <header className="masthead bg-primary text-white text-center">
          <div className="container d-flex align-items-center flex-column">
            <h1 className="masthead-heading text-uppercase mb-0">Kahoot! Clone</h1>
            <div className="divider-custom divider-light">
              <div className="divider-custom-line"></div>
              <div className="divider-custom-icon">
                <i className="fas fa-star"></i>
              </div>
              <div className="divider-custom-line"></div>
            </div>
            <div className="masthead-subheading font-weight-light mb-0">
              <Link to="/host/chooseTrivia" className="btn">
                <Button className="join-host" variant="secondary">
                  I want to host a Trivia
                </Button>
              </Link>
              <Link to="/user" className="btn">
                <Button className="join-user" variant="secondary">
                  I want to join a Trivia
                </Button>
              </Link>
              <Link to="/admin/stats" className="btn">
                <Button className="join-user" variant="info">
                  Statistics
                </Button>
              </Link>
            </div>
          </div>
        </header>
      </body>
    </React.Fragment>
  );
};

export default Home;
