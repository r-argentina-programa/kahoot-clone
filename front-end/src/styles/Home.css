.containerButtons {
  display: flex;
  justify-content: space-evenly;
}
