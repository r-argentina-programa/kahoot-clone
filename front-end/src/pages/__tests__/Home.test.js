import Home from '../Home';
import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Home />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
