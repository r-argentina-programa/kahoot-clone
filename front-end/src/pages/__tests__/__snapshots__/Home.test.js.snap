// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container h-center"
>
  <a
    href="/host"
    onClick={[Function]}
  >
    <button
      className="join-host btn btn-primary"
      disabled={false}
      type="button"
    >
      I want to host a Trivia
    </button>
  </a>
  <a
    href="/user"
    onClick={[Function]}
  >
    <button
      className="join-user btn btn-primary"
      disabled={false}
      type="button"
    >
      I want to join a Trivia
    </button>
  </a>
  <a
    href="/admin/stats"
    onClick={[Function]}
  >
    <button
      className="join-user btn btn-primary"
      disabled={false}
      type="button"
    >
      Statistics
    </button>
  </a>
</div>
`;
